config/machine_readable/register-to-vote.yml

```diff
@@ -1,57 +1,59 @@
-        <li>a UK citizen (or an Irish, EU or Commonwealth citizen with a permanent UK address)</li>
+        <li>a British citizen</li>
+        <li>an Irish or EU citizen living in the UK</li>
+        <li>a Commonwealth citizen who has permission to enter or stay in the UK, or who does not need permission</li>
+        <li>a citizen of another country living in Scotland or Wales who has permission to enter or stay in the UK, or who does not need permission</li>
       </ul>
-      <p>You will not be able to vote until you’re 18. If you live in Scotland, you can vote in Scottish Parliament and local elections when you’re 16.</p>
+      <p>Check which <a href="/elections-in-the-uk?src=schema">elections you’re eligible to vote in</a>.</p>
 
   - question: Registering online
     answer: >
-      <p>Use this service to register to vote.</p>
-      <p>You only need to register once - not for every election.</p>
+      <p>It usually takes about 5 minutes.</p>
       <p><a rel="external" href="https://www.registertovote.service.gov.uk/register-to-vote/start?src=schema">Start now</a></p>
       <h2>What you need to know</h2>
       <p>You’ll be asked for your National Insurance number (but you can still register if you do not have one).</p>
+      <p>After you’ve registered, your name and address will appear on the electoral register.</p>
 
-  - question: How to check if you’re already registered
+  - question: Check if you’re already registered
     answer: >
-      <p>Contact your local Electoral Registration Office to <a href="/get-on-electoral-register?src=schema">find out if you’re already registered to vote</a>.</p>
+      <p><a href="/contact-electoral-registration-office?src=schema">Contact your local Electoral Registration Office</a> to find out if you’re already registered to vote.</p>
 
-  - question: Updating your registration
+  - question: Update your registration
     answer: >
       <p>You can also use the <a rel="external" href="https://www.registertovote.service.gov.uk/register-to-vote/start?src=schema">‘Register to vote’ service</a> to:</p>
       <ul>
         <li>change your name, address or nationality</li>
-        <li>get on or off the <a href="/electoral-register?src=schema">open register</a>
-      </li>
+        <li>get on or off the <a href="/electoral-register?src=schema">open register</a></li>
       </ul>
       <p>To do this, you need to register again with your new details (even if you’re already registered to vote).</p>
 
-  - question: Registering with a paper form
+  - question: Register with a paper form
     answer: >
       <p>You can:</p>
       <ul>
         <li><a href="/government/publications/register-to-vote-if-youre-living-in-the-uk?src=schema">register using a paper form in England, Wales and Scotland</a></li>
         <li><a rel="external" href="http://www.eoni.org.uk/Register-To-Vote/Register-to-vote-change-address-change-name?src=schema">register using a paper form in Northern Ireland</a></li>
```

Comment:
Non-blocker: we can now use HTTPS here `https://www.eoni.org.uk/Register-To-Vote/Register-to-vote-change-address-change-name?src=schema` and in all the eoni links.